build/common

```diff
@@ -280,6 +280,9 @@ cleanup() {
         cp ${dir}/root/ids.tables ${dir}/../ids.tables
     fi
 
+    # Verify the IDS are correct
+    [ -f "${dir}/usr/sbin/check-ug.py" ] && do_chroot ${dir} /usr/sbin/check-ug.py
```

Comment:
you need to run the script only when packaging tool is RPM, otherwise it will make Debian/Ubuntu builds failing.